examples/rapids/mlflow_project/README.md

```diff
@@ -0,0 +1,47 @@
+### Train and Publish Locally With RAPIDS and MLflow
+**[RAPIDS](https://rapids.ai/)** is a suite of open source libraries for GPU-accelerated analytics. 
+
+**[RAPIDS cuML](https://github.com/rapidsai/cuml)** matches the scikit-learn API, so it can build on MLflow's existing support for scikit-learn-like models to support
+persistence and deployment."
+
+The example workflows below train RAPIDs regression models to predict airline flight delays, using MLflow to log models and deploy them as local REST API endpoints for real-time inference.
+
+#### Jupyter Notebook Workflow
+[Jupyter Notebook](notebooks/rapids_mlflow.ipynb)
+
+#### CLI Based Workflow
+1. Download data
+    1. `cd examples/rapids/mlflow_project`
+        ```shell script
+        # Download the file
+        wget -N https://rapidsai-cloud-ml-sample-data.s3-us-west-2.amazonaws.com/airline_small.parquet
+        ```
+1. Set MLflow tracking uri
+    1. ```shell script
+        export MLFLOW_TRACKING_URI=sqlite:////tmp/mlflow-db.sqlite
+       ```
+1. Train the model using a single run.
+    1. ```shell script
+       # Launch the job
+       mlflow run . -e simple\
+                --experiment-name RAPIDS-CLI \
+                -P max_depth=10 -P max_features=0.75 -P n_estimators=500 \
+                -P conda-env=$PWD/envs/conda.yaml \
```

Comment:
We no longer need to pass conda-env here right?
```suggestion
```

Also, I like the idea of having an easy-to-run "simple" entry point - could we add default values for the max_depth, max_features, n_estimators params in the MLproject file, as shown in [the docs](https://mlflow.org/docs/latest/projects.html#mlproject-file), and mention that the user can simply run `mlflow run . -e simple`?